config.yaml

```diff
@@ -15,3 +15,9 @@ version: 1
 target_user: root
 
 #event_cache_size: 3
+
+# maximum age of an artifact folder in days
```

Comment:
I will add:
# set to 0 to disable the automatic removal of old artifact folders